### PR TITLE
Bitrunning: Tweaks, QoL and removals

### LINKED ISSUE #83787: Wild West bitrunning sim takes extremely long to initialize, and is unusable until it does
Reporting client version: 515.1637

## Issue Summary
When loading the Wild West/Gunslinger Alley/Etc bitrunning sim (the one with the ULTRAKILL guns), it takes an extremely long time for the map to initialize - Random spawners to spawn, lights to generate and propagate, and importantly, atmos to be filled. The sim has no air until it finishes

## Round ID:
[230147](https://scrubby.melonmesa.com/round/230147)

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Incoming stamina damage while in stamcrit has diminishing returns applied](https://www.github.com/tgstation/tgstation/pull/83445)
- [Adds some logging to roundstart, to figure out why it's been taking longer recently](https://www.github.com/tgstation/tgstation/pull/83745)
- [Cult Vs. Heretic: 7 Months Later Edition](https://www.github.com/tgstation/tgstation/pull/82877)

## Reproduction:
Be a GAMER
Load your favorite gunslinger-fantasy HARD bitrunning sim
Screenshot 2024-06-07 115143

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request

See changelog for shortlist

1. **Threat changes.** I was a bit unsatisfied with the rate of antag spawns. These have been increased considerably. The clamped probability has been increased from 1-10 to 5-15. The probability increases from 5 to 15 as domains are completed. Generally, in a standard round, the chance of spawning at least one antag should be around ~50% at 7 domains completed. Emagging a server doubles this rate.
2. **Map changes.** Starfront saloon was a cool idea on paper: A totally modular map. However, it looked very uninspired and was so much of a chore on the map loading system that it prompted players to admin help how long it took, thinking it was broken. I've removed the map. I have others I want to implement that don't look so bad.
3. **QoL changes**. Ghost observer experience is improved. Previously, you could click netpods to view their avatar, and now you can click the hololadder to return. I've included examine text to show this. The server's examine text will now also give you clues that it's emagged (ghost only). The examine text on hololadders has also been improved.
4. **Bitrunning antags.** These were designed as temporary, but they were everything but. Spawning as one would prevent your revival, which just isn't a good tradeoff for something that's going to get deleted in a minute. Now, this system uses temp bodies just like CTF, so you can return once you're dead. (exception: coming station side)
5. **Maps**: Syndicate assault is still one of my favorites, but there's cheesy exploits like instantly breaking the display case to lock down the ship, turning on turrets which are EXTRA lethal, etc. I've added some pistols to the closets and removed some of these exploits.
6. **Cooldown**: Yes, no one seems to upgrade these ever, and it proved a poor technique to encourage bitrunners to leave their rooms. I had other plans to encourage this, not included here, so I think lowering the cooldown time is beneficial. 3min -> 2min

> [!NOTE]
> File diff: removed a map

## Why It's Good For The Game
Closes #83787
General updates and QoL for bitrunning to keep it fresh. I was quite disappointed with the scaling of threat, and most players haven't even seen bitrunning antags except when I admin spawn them. These numbers aren't hard set in my mind, and could be adjusted.

I generally want bitrunning easier to access and more "temporary" which is in keeping with its design doc.
## Changelog
:cl:
fix: Bitrunning made more illegal: Increased the rate at which antags spawn.
fix: "Temporary" bitrunning antagonists and spawners are made actually temporary. You will return to your original body after death, just like CTF.
add: Added more examine text for ghosts to bitrunning equipment.
balance: Server cooldown reduced by 1 minute at base level.
add: As an observer, you can now switch views between station and virtual domain by clicking the hololadder and netpod respectively.
del: Removed the starfront saloon BR map.
fix: Syndicate assault map: Added pistols, reduced exploits.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
if i can ideasguy maps that let you play as antags or fight against player antags would br great to let people experience uncommon antags in a less stressful environment

**Comment by jlsnow301** [comment]
> if i can ideasguy maps that let you play as antags or fight against player antags would br great to let people experience uncommon antags in a less stressful environment

good idea and very possible to do. we really need some easy tier maps, and these could be tutorials for certain gameplay elements (abductors?)

**Comment by Loafin34** [comment]
what if you had a revolution simulator that could summon 8 weakly-equipped player antags against one or two bitrunners with sec gear?

**Comment by OuroborusReporter** [comment]
Rest in piss rootin shootin shootin sim, you'll be missed
This closes #83787

**Comment by SimplySmiley123** [comment]
> what if you had a revolution simulator that could summon 8 weakly-equipped player antags against one or two bitrunners with sec gear?

Honestly yeah.
Just security gear for bitrunners in their safehouse vs. 8-10 player controlled assistants.
Tho that would probably rarely ever work fully. 1 or 2 ghosts taking over will often happen, but any more then that is asking for a lot.

**Comment by jlsnow301** [comment]
> Honestly yeah. Just security gear for bitrunners in their safehouse vs. 8-10 player controlled assistants. 

This has been a long standing idea I wanted, greytide brawl, and I do have the map for it already... Just need to code it

**Comment by jlsnow301** [comment]
Hopefully good changes, check the PR desc @ 4

File: code/modules/bitrunning/event.dm [hunk 1]
```diff
@@ -89,6 +89,10 @@
 	if(!unlucky_server.validate_mutation_candidates())
 		return WAITING_FOR_SOMETHING
 
-	spawned_mobs = unlucky_server.setup_glitch(forced_role)
```
Comment (jlsnow301) [inline code/modules/bitrunning/event.dm:92]: this was causing a runtime (looks for a list, not a mob)

**Pull request merged**